### PR TITLE
move to uplink acapy client

### PR DESCRIPTION
**Pull request by TimoGlastra**
Signed-off-by: Timo Glastra <timo@animo.id>

- [x] Fix tests for those which aren't working
- [x] Fix multi-tennant endpoints
- [x] Fix pubDID
- [x] release cloudcontroller "final" version (working for what we have) and pull that in requirements via pypi instead of from a git branch/commit

### CODE REVIEW

File: app/generic/connections.py [hunk 1]
```diff
@@ -10,37 +10,38 @@
...
+    invite = await aries_controller.connection.create_invitation()
     return invite
 
 
-@router.post("/accept-invite")
+@router.post("/accept-invite", response_model=ConnRecord)
 async def accept_invite(
-    invite: dict,
-    aries_controller: AriesAgentControllerBase = Depends(agent_selector),
+    invite: ReceiveInvitationRequest,
```
Comment (TimoGlastra) [inline app/generic/connections.py:26]: Input models can also be specified based on the cloud controller models

**Review by TimoGlastra — COMMENTED** [review]
I've updated the connections endpoints to use the generated cloud controller and added some comments for context.

**Comment by TimoGlastra** [comment]
I've updated the cloud controller to 0.3.0a7, which contains some fixes:
- Updated the OpenAPI with some manual fixes. I'll see later to get those into ACA-Py itself (mainly ledger operations)
- Added support for anyOf / oneOf models

**Comment by TimoGlastra** [comment]
@morrieinmaas @vineeth14 Have you run into any issues with using the cloud controller?

**Review by morrieinmaas — APPROVED** [review]

**Comment by morrieinmaas** [comment]
@TimoGlastra I made the tests pass now and removed some code smells. There is one test that needs fixing which is `test_send_credential_proposal` (for that please see [this issue in the controller](https://github.com/didx-xyz/aries-cloudcontroller-python/issues/62)). 

Please also have a look at issues 55-61 in the cloudcontroller. I've tweaked the controller for those issue on am now installing the controller from a branch via git+https://github.com/didx-xyz/aries-cloudcontroller-python@1162bb8b51ef5ddcaec916708c66966e3f18c358 as dependency. Before we we fully move to uplink version of the controller I'd say we should fix the issues and return types. That said, as you'll see in the code replacing a PyDantic model with an ANy type returns a couroutine object instead of a model object so I had to await it's JSOn response seprately. That is not ideal either, but works for now and will disappear when types are fixed. 

Do NOT merge yet.

**Comment by morrieinmaas** [comment]
closes #13 and closes #59

**Pull request merged**